src/MetalPerformanceShaders/MPSDefs.cs

```diff
@@ -74,14 +75,30 @@ public enum MPSDataType : uint { // uint32_t
 		Unorm8 = NormalizedBit | 8,
 	}
 
-	[iOS (10,0)][TV (10,0)][Mac (10, 13)]
+	[Flags]
+	[Native]
+	public enum MPSAliasingStrategy : ulong
```

Comment:
I realize it's a backport (I seem to have missed the PR) but this, and others, seems to be missing availability attributes